app/helpers/cases_helper.rb

```diff
@@ -27,6 +27,11 @@ def action_button_for(event)
               respond_case_path(@case),
               id: 'action--mark-response-as-sent',
               class: 'button'
+    when :mark_response_sent_and_close
+      link_to t('common.case.mark_response_sent_and_close'),
+              respond_case_path(@case),
+              id: 'action--mark-response-as-sent-and-close',
+              class: 'button'
```

Comment:
I _want_ to say "AAHHH DON'T ADD ANYTHING TO THIS METHOD USE ACTION_LINK* BELOW AAAGGHHGHH!!!" but ... I kind of just want to see this get merged. So if you see me sighing morosely for a bit ... that's why.